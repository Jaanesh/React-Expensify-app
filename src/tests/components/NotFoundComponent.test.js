import React from 'react';
import {shallow} from 'enzyme';
import NotFoundComponentTemplate from '../../components/NotFoundComponent.js';

test('should render NotFoundComponent',()=>{
       const wrapper=shallow(<NotFoundComponentTemplate/>);
       expect(wrapper).toMatchSnapshot();
});