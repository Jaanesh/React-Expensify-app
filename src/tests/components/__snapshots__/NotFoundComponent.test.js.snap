// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundComponent 1`] = `
<div>
  404 
  <Link
    replace={false}
    to="/dashboard"
  >
    GoHome
  </Link>
</div>
`;
